src/components/ui/tab-panel-tab-content.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import clsx from "clsx";
+
+interface ITabPanelTabContentProps {
+  index: number;
+  selectedIndex: number;
+  disabled: boolean;
+  children: React.ReactNode
+}
+
+export const TabPanelTabContent = ({index, selectedIndex, disabled, children}: ITabPanelTabContentProps) => {
+  const selected = index === selectedIndex;
+  const hidden = !selected;
+  const className = clsx("ui-item ui-multiview-item", {
+    "ui-item-selected": selected,
+    "ui-multiview-item-hidden": hidden,
+    "ui-state-disabled": disabled,
+  })
+
+  return (
+    <div
+      className={className}
+      role="tabpanel"
+      style={{transform: "translate(0px, 0px)"}}
```

Comment:
This and the similar translate value in tab-panel.tsx confused me a little. Do you have to apply this here to override a transform defined in the stylesheet? Would it make sense to use the `ui-translate-disabled` class instead?

Reply:
This mirrors the output of the `devextreme` tab panel output.  Basically I just copied the prop types of the components source and the outer html of each rendered component and worked backwards to create a component that mirrored that.